scripts/run-cypress-container.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+###########################################################################
+# Local dev script for testing cypress e2e tests in container with Podman #
+###########################################################################
+
+set -eoux pipefail
+
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+DOCKER_TAG="$(git rev-parse --short HEAD)"
+app=$1
+spec=$2
+
+APP_HOME=$(jq ".projects[\"$app\"]" -r < "$PROJECT_ROOT"/workspace.json)
+APP_DIST_HOME=$(jq ".targets.build.options.outputPath" -r < "$PROJECT_ROOT"/"$APP_HOME"/project.json)
+APP_SPEC=$(jq ".targets[\"$spec\"].options.spec" -r < "$PROJECT_ROOT"/"$APP_HOME"/project.json)
+
+export DOCKER_TAG="${DOCKER_TAG}"
+
+podman run \
```

Comment:
docker should be the default here. Maybe keep this file out of the PR for now to avoid slowing it down.

Reply:
my local test file, should not be there